frontend/catalyst/pennylane_extensions.py

```diff
@@ -26,6 +27,7 @@
 import jax
 import jax.numpy as jnp
 import pennylane as qml
+import toml
```

Comment:
@erick-xanadu I know the PR is still a draft :) I just wanted to double check now, to avoid more work in the future: @mlxd, is this the TOML library we have chosen to go with?

Reply:
We choose `tomllib`  to implement the support from [ADR 73](https://github.com/PennyLaneAI/adrs/blob/master/documents/073-catalyst-devices-communication.md) as this is the default toml package in Python 3.11 and is supported  in other versions of Python. I've added the details of how to use tomllib and why?! to that ADR.

Reply:
Thanks Ali! My fault, I forgot we had already discussed this 🤦

Reply:
I am now using the above strategy.

Reply:
@maliasadi a question here: if TOML reading is now a requirement to using Catalyst, should this not be a build dependency?

Reply:
Ah, https://github.com/PennyLaneAI/catalyst/commit/d9f23c7b83f33bdff5d65df5a34460cf04f12134 should resolve this :)